data/remotes.d/vendor.conf

```diff
@@ -4,4 +4,5 @@
 # /usr/share/fwupd/remotes.d/vendor and /usr/share/fwupd/remotes.d/vendor/firmware
 Enabled=false
 Type=local
+Keyring=none
```

Comment:
I think it would still be a good practice to encourage signatures even when metadata is stored locally on the box/image in an embedded scenario.  Maybe just the README needs to mention this.